go-controller/pkg/ovn/base_network_controller_multipolicy_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	netv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/k8snetworkplumbingwg/multi-networkpolicy/pkg/apis/k8s.cni.cncf.io/v1beta1"
+	"github.com/k8snetworkplumbingwg/multi-networkpolicy/pkg/apis/k8s.cni.cncf.io/v1beta2"
```

Comment:
Maybe we should change this import to be named, like `mnpapi`, so we could avoid extra changes in the code in case we need to bump the version again?

Reply:
done